website/docs/using-podman-desktop-behind-a-proxy/index.md

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_position: 3
+title: Using behind a proxy
+description: Using Podman Desktop behind a proxy requiring custom Certificate Authorities (CA).
+tags: [podman-desktop, installing, windows, macos, linux]
+keywords: [podman desktop, containers, podman, installing, installation, windows, macos, linux]
+---
+
+# Using Podman Desktop behind a proxy requiring custom Certificate Authorities (CA)
```

Comment:
This is a false positive:

> [RedHat.Headings] Use sentence-style capitalization in 'Using Podman Desktop behind a proxy requiring custom Certificate Authorities (CA)'.

* Reported: https://github.com/redhat-documentation/vale-at-red-hat/issues/455
* Proposed a fix: https://github.com/redhat-documentation/vale-at-red-hat/pull/456